### PR TITLE
feat[tool/aggregator]: using verifyMixedAggregatedProof

### PR DESCRIPTION
**Pull request by re1ro**
- [x] `tool/aggregator`, `tool/devAggregator`: utilizing verifyMixedAggregatedProof
- [x] `computeAggregatedProofParameters`: adding `numOffChainProofs` and removing `proofIds`

### CODE REVIEW

File: upa/src/tool/devAggregator.ts [hunk 1]
```diff
@@ -20,6 +20,13 @@ import {
   packOffChainSubmissionMarkers,
 } from "../sdk/aggregatedProofParams";
 
+type SubmissionIntervalWithMetadata = SubmissionInterval<
+  | {
```
Comment (dtebbs) [inline upa/src/tool/devAggregator.ts:24]: Fine to add support here, but just to check - the dev-aggreagtor isn't yet able to read off-chain submissions, presumably?

**Review by dtebbs — APPROVED** [review]
1 question, otherwise LGTM.  Will leave final call to @akileshtangella

File: upa/src/tool/aggregator.ts [hunk 2]
```diff
@@ -166,176 +166,185 @@ const computeFinalDigest = command({
...
+      );
+      const optionsPayable: PayableOverrides = {
+        maxFeePerGas: utils.parseGweiOrUndefined(maxFeePerGasGwei),
+      };
+      const txReq = await verifier[
+        mixed ? "verifyMixedAggregatedProof" : "verifyAggregatedProof"
+      ].populateTransaction(
+        calldata,
+        proofIds,
+        apParams.numOnChainProofs,
```
Comment (re1ro) [inline upa/src/tool/aggregator.ts]: If mixed then `apParams.numOffChainProofs`

File: upa/src/tool/aggregator.ts [hunk 3]
```diff
@@ -403,7 +412,8 @@ export const aggregator = subcommands({
     "allocate-aggregator-fee": allocateAggregatorFee,
     "claim-aggregator-fee": claimAggregatorFee,
     "compute-final-digest": computeFinalDigest,
-    "submit-aggregated-proof": submitAggregatedProof,
+    "submit-aggregated-proof": makeSubmitAggregatedProofCommand(false),
```
Comment (re1ro) [inline upa/src/tool/aggregator.ts]: Creare a union type `mixed | on-chain` instead of true/false

File: upa/src/tool/devAggregator.ts (hunk 1 above)
Comment (re1ro) [reply upa/src/tool/devAggregator.ts:24]: Yeah it currently reads only from on-chain events. But I added general support for off-chain handling. It defaults to `undefined` which is falsy.

We could add reading offchain submissions from file later

**Review by akileshtangella — APPROVED** [review]
LGTM.

**Review by dtebbs — APPROVED** [review]

**Pull request merged**